resources/views/individuals/partials/progress.blade.php

```diff
@@ -39,7 +39,12 @@
                         namespace="publish_page" />
                 @else
                 <p id="cannot-publish-explanation">
-                    {{ __('You must attend an orientation session and fill in all the required information before you can publish your page.') }}
+                    {!! Str::markdown(
+                        __(
+                            'You must attend an [orientation session](:url) and fill in all the required information before you can publish your page.',
+                            ['url' => orientation_link('individual')],
```

Comment:
It's better not to hardcode the argument value. First attempt to get it from something passed into the page, for example from the User. If that isn't possible you should get it via the enum (e.g. `orientation_link(App\Enums\UserContext::Individual->value)`). That will minimize the maintenance needed for name changes.

Reply:
@chosww you could probably if that blade file is only ever used for regulated organizations. Otherwise pulling the context from the User would work as well because it should use the enum value internally.